README.fi.md

```
@@ -0,0 +1,502 @@
+5. Click the **Authorize** button.
+   ![step4](docs/images/openrouter/4.png)
+
+6. Prepay as much as you plan to use.
+   ![step5](docs/images/openrouter/5.png)
+
+<details>
+<summary><h3>Jos Authorize-painike ei toiminut</h3></summary>
+
+Jos klikkasit Authorize mutta et ole edelleen tunnistautunut, yritä uudelleen tai luo API-avain manuaalisesti:
+
+6. Click your account in the top right, go to the API Keys tab on the left, then click the Create button in the center.
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 6; Style: 1/2/3

(MD029, ol-prefix)